tardis/montecarlo/montecarlo.pyx

```diff
@@ -18,6 +18,14 @@ np.import_array()
 
 ctypedef np.int64_t int_type_t
 
+cdef extern from "numpy/arrayobject.h":
+    void PyArray_ENABLEFLAGS(np.ndarray arr, int flags)
+
+cdef c_array_to_numpy(void *ptr, int dtype, np.npy_intp N):
```

Comment:
This is just a really small thing, but as we are using numpy API, I think we should use `np.int_t dtype` instead of `int dtype`.